tern/analyze/common.py

```diff
@@ -53,6 +53,7 @@ def load_from_cache(layer, redo=False):
             layer.os_guess = cache.cache[layer.fs_hash]['os_guess']
             layer.pkg_format = cache.cache[layer.fs_hash]['pkg_format']
         load_files_from_cache(layer)
+
```

Comment:
There's an extra line here. You just need to reset this file or remove the line.